tests/components/renault/snapshots/test_binary_sensor.ambr

```diff
@@ -1005,102 +1005,6 @@
     'state': 'off',
   })
 # ---
-# name: test_binary_sensors[twingo_3_electric][binary_sensor.reg_twingo_iii_driver_door-entry]
```

Comment:
Snapshots and tests updated due to https://github.com/hacf-fr/renault-api/pull/1567